hazelcast/src/main/java/com/hazelcast/nio/serialization/GenericRecord.java

```diff
@@ -28,13 +30,13 @@
 import java.util.Set;
 
 /**
- * A generic object interface that is returned to user when the domain class can not be created from any of the distributed
- * hazelcast data structures like {@link com.hazelcast.map.IMap} ,{@link com.hazelcast.collection.IQueue} etc.
+ * A generic object interface that is returned to the user when the domain class can not be created from any of the distributed
```

Comment:
I don't see how the javadoc changes and the missing connection to participant relate. Should this be split into two separate PRs?

Reply:
Yes, it should, but I was lazy. I'm fixing grammar as I read it, and then submit all the collected changes in the next _real_ PR i'm doing.